src/main/java/uk/gov/hmcts/ccd/domain/service/casedataaccesscontrol/PseudoRoleAssignmentsGenerator.java

```diff
@@ -38,12 +38,13 @@ public PseudoRoleAssignmentsGenerator(@Qualifier(CachedUserRepository.QUALIFIER)
         this.caseAccessService = caseAccessService;
     }
 
-    public List<RoleAssignment> createPseudoRoleAssignments(List<RoleAssignment> filteredRoleAssignments) {
+    public List<RoleAssignment> createPseudoRoleAssignments(List<RoleAssignment> filteredRoleAssignments,
+                                                            boolean isCreationProfile) {
 
         List<String> idamUserRoles = new ArrayList<>(userRepository.getUserRoles());
         List<RoleAssignment> pseudoRoleAssignments = new ArrayList<>();
 
-        if (caseAccessService.userCanOnlyAccessExplicitlyGrantedCases()) {
+        if (!isCreationProfile && caseAccessService.userCanOnlyAccessExplicitlyGrantedCases()) {
```

Comment:
caseAccessService.userCanOnlyAccessExplicitlyGrantedCases() --> instead of this can we use 
UserAuthorisation.AccessLevel.GRANTED.equals(userAuthorisation.getAccessLevel())